src/crypto/CHIPCryptoPAL.cpp

```diff
@@ -419,7 +421,9 @@ CHIP_ERROR Spake2p::GenerateKeys()
 {
     static const uint8_t info_keyconfirm[16] = { 'C', 'o', 'n', 'f', 'i', 'r', 'm', 'a', 't', 'i', 'o', 'n', 'K', 'e', 'y', 's' };
 
-    ReturnErrorOnFailure(HashFinalize(Kae));
+    MutableByteSpan Kae_span{ &Kae[0], sizeof(Kae) };
```

Comment:
```suggestion
    MutableByteSpan Kae_span { Kae };
```
likely works too.... either way.

Reply:
Since it's not in local scope to see what Kae is, I tried to make the code very explicit that this is a bytespan over an array far away.